barebones/ee_printf.c

```diff
@@ -29,9 +29,9 @@ limitations under the License.
 
 static char *digits = "0123456789abcdefghijklmnopqrstuvwxyz";
 static char *upper_digits = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ";
-static ee_size_t strnlen(const char *s, ee_size_t count);
```

Comment:
What's the difference between ee_size_t and size_t?

Reply:
You're right, there is no difference, good catch I'll change it back